tests/test_mdspan_size.cpp

```diff
@@ -44,12 +44,12 @@ void test_mdspan_size(std::vector<char>& storage, Extents&& e)
   using extents_type = std::remove_cv_t<std::remove_reference_t<Extents>>;
   Kokkos::mdspan<char, extents_type> m(storage.data(), std::forward<Extents>(e));
 
-  static_assert(std::is_same<decltype(m.size()), std::size_t>::value,
-		"The return type of mdspan::size() must be size_t.");
+  static_assert(std::is_same<decltype(m.size()), typename decltype(m)::size_type>::value,
+		"The return type of mdspan::size() must be size_type.");
```

Comment:
Shouldn't we update the type of min_storage_size from size_t to size_type then?

Reply:
Sure